sklearn/feature_extraction/hashing.py

```diff
@@ -41,10 +41,12 @@ class FeatureHasher(BaseEstimator, TransformerMixin):
         The number of features (columns) in the output matrices. Small numbers
         of features are likely to cause hash collisions, but large numbers
         will cause larger coefficient dimensions in linear learners.
+
```

Comment:
Do you realise, btw, that other numpydoc projects do not have these spaces between parameter descriptions?